worlds/animal_well/names.py

```diff
@@ -463,3 +464,120 @@ def __str__(self) -> str:
+    fruit_34 = "Pink Fruit in Capybara Room Bottom Left - R4C15d"  # pink fruit bottom left
+    fruit_35 = "Blue Fruit Above Plant Egg Chest - R5C1"  # blue fruit
+    fruit_36 = "Pink Fruit Below Dog Many Switches Room - R5C3"
+    fruit_37 = "Pink Fruit Before Mock Disc Chest - R5C4"
+    fruit_38 = "Pink Fruit by Phone Left of Mock Disc Chest - R5C6"
+    fruit_39 = "Blue Fruit by Dog Wheel - R5C9"  # blue fruit
+    fruit_40 = "Blue Fruit Above Bear Kangaroo Waterfall - R5C11"  # blue fruit
+    fruit_41 = "Blue Fruit by Universal Basic Egg Chest - R5C13"  # blue fruit
+    fruit_42 = "Pink Fruit in Dog Dark Room - R6C5"
+    fruit_43 = "Blue Fruit After Dachshund Tunnels - R6C9a"  # blue fruit
+    fruit_44 = "Pink Fruit Above Egg Room - R6C9b"
+    fruit_45 = "Pink Fruit Before Truth Egg - R6C10a"
+    fruit_46 = "Pink Fruit Bottom of Dog Elevator - R6C10b"
+    fruit_47 = "Pink Fruit by Bear Kangaroo Waterfall - R6C11"
+    fruit_48 = "Blue Fruit After First Chameleon Fight - R6C12"  # blue fruit
+    fruit_49 = "Blue Fruit in Bear Dark Maze Right - R6C14a"  # blue fruit, verified
+    fruit_50 = "Pink Fruit in Bear Dark Maze Left - R6C14b"
+    fruit_51 = "Pink Fruit Below Capybara Room - R6C15"
+    fruit_52 = "Pink Fruit in Chinchilla Vines Room - R6C16"
+    fruit_53 = "Blue Fruit by Ancient Egg Chest Right - R7C1a"  # blue fruit, verified
+    fruit_54 = "Pink Fruit by Ancient Egg Chest Left - R7C1b"
+    fruit_55 = "Pink Fruit in Fish Pipe Maze Left - R7C2"
+    fruit_56 = "Blue Fruit in Fish Pipe Maze Right - R7C3a"  # blue fruit, verified
+    fruit_57 = "Pink Fruit in Fish Pipe Maze Middle - R7C3b"
+    fruit_58 = "Pink Fruit by Fireball Thrower - R7C4"
+    fruit_59 = "Pink Fruit by Flute Chest - R7C6"
+    fruit_60 = "Blue Fruit Behind Entry Chameleon - R7C14"  # blue fruit
+    fruit_61 = "Pink Fruit in Bear Entry Room - R7C15"
+    fruit_62 = "Pink Fruit by Lower Bear Key Chest - R7C16"
+    fruit_63 = "Pink Fruit Above Fish Fast Travel - R8C1"
+    fruit_64 = "Pink Fruit in Fish Upper Phone Room Top - R8C2a"
+    fruit_65 = "Pink Fruit in Fish Upper Phone Room Bottom - R8C2b"
+    fruit_66 = "Pink Fruit Left of Hub Left - R8C9a"
+    fruit_67 = "Pink Fruit Left of Hub Right - R8C9b"
+    fruit_68 = "Pink Fruit by Mural Bunny - R813"
+    fruit_69 = "Pink Fruit by Fish Fast Travel - R9C1"
+    fruit_70 = "Pink Fruit Below Fish Upper Phone Room Top - R9C2"
+    fruit_71 = "Pink Fruit Before First Seahorse Fight - R9C3a"
+    fruit_72 = "Blue Fruit Below Fish Upper Phone Room Bottom - R9C3b"  # blue fruit, verified
+    fruit_73 = "Pink Fruit Bottom Left of Fish Bubble Column - R9C4"
+    fruit_74 = "Pink Fruit Bottom Right of Fish Bubble Column - R9C5"
+    fruit_75 = "Big Blue Fruit After Flamingos - R9C7"  # big blue fruit, right of flamingos
+    fruit_76 = "Pink Fruit by Water Spike Bunny - R9C9"
+    fruit_77 = "Pink Fruit Below Mural Bunny Left - R9C13a"
```

Comment:
This is not "below mural bunny" in my head. "below mural bunny", if anything, is R9C12/R10C12, which has its own fruit in it already.
```suggestion
    fruit_77 = "Pink Fruit by Planet Egg Chest Left - R9C13a"
```